app/views/course/assessment/answer/_worksheet.html.slim

```diff
@@ -2,7 +2,8 @@
   - if answer.attempting?
     div.btn-group
       = link_to_reset_answer(answer)
-      = base_answer_form.button :button, t('common.submit'), value: answer.id, name: 'attempting_answer_id', class: 'submit-answer'
+      = base_answer_form.button :button, t('common.run_code'), value: answer.id,
```

Comment:
@justwudi Small issue, 'Run Code' should not be put in the common translation,  use `t('.run_code')` here and put the translation in the appropriate place ?